### ISSUE TITLE
account encryption tests are failing on ARM

### ISSUE THREAD

**Issue by splix** [ethereumproject/go-ethereum#45]
Following tests are failing on ARM, without giving any stacktrace:
- accounts.TestV1_2  
- accounts.TestV3_PBKDF2_1  
- accounts.TestV3_PBKDF2_2  
- accounts.TestV3_PBKDF2_3  
- accounts.TestV3_PBKDF2_4  
- accounts.TestV3_Scrypt_1  
- accounts.TestV3_Scrypt_2  
- accounts.TestWatchNoDir 

http://build.ethereumclassic.org/viewLog.html?buildId=252&tab=buildResultsDiv&buildTypeId=GoEthereumClassic_Develop_TestArm

**Comment by ericsomdahl** [comment]
I investigated this:  digging further into the logs I found many errors like this:
/usr/local/go/pkg/tool/linux_arm/link: running gcc failed: fork/exec /usr/bin/gcc: cannot allocate memory

Perhaps there is a memory leak in the go ARM toolchain, or in the ARM gcc build.

I rebooted the RPi and kicked off another test build.  It may be the case that the test suite cannot fully run on ARM.  The upstream full test suite is only run against the Linux platform.
